### ISSUE TITLE
[themes] add `high contrast light themes` separator

### ISSUE THREAD

**Issue by vince-fugnitto** [microsoft/vscode#166716]
#### Description

The proposal is to add the additional `high contrast light themes` or `light high contrast themes` separator to the themes quick-pick for accessibility reasons, and consistent. As a consequence the existing `high contrast themes` should be renamed to `high contrast dark themes` or `dark high contrast themes` respectfully.

The main argument is to more accurately group themes similarly to the `light themes` and `dark themes` groups, and better differentiate between user installed themes. At the moment all different types of high contrast themes are grouped under a common separator which can make their discovery more difficult as they will only be in alphabetical order.

https://github.com/microsoft/vscode/blob/2fe850b690fd4e9f53c86055fa7aedd825800ea6/src/vs/workbench/contrib/themes/browser/themes.contribution.ts#L386-L390

_Before_:

![image](https://user-images.githubusercontent.com/40359487/202753400-0588be67-d133-4171-922a-ad37f89602a2.png)

_After_:

![image](https://user-images.githubusercontent.com/40359487/202757936-63dca625-d91c-4212-b076-9ced597522d0.png)

**Comment by daviddossett** [comment]
If we had multiple examples of each theme type, I could see this making sense. But given that we only ship one of each, the categories seem like they add a lot of noise for the vast majority of users that won't have more than the two that ship out of the box.

**Comment by vince-fugnitto** [comment]
@daviddossett yea, I'm more so talking about when multiple user-installed themes exist for both categories, in that case we may want to better group them. The default as you mentioned could be to display the common "high contrast themes" until a certain threshold is met to facilitate their discovery.

**Comment by vince-fugnitto** [comment]
@daviddossett I'm not sure it adds much noise, there are other menus such as `open view` that add much more noise than something like selecting a theme would (granted it provides information about location):

![image](https://user-images.githubusercontent.com/40359487/202770865-ddb4b77b-8d75-4f72-a447-3fef3788c216.png)

**Comment by aeschli** [comment]
I kind of like the current way. I'd keep things unless we get more votes for this

**Comment by VSCodeTriageBot** [comment]
This feature request is now a candidate for our backlog. The community has 60 days to [upvote](https://github.com/microsoft/vscode/wiki/Issues-Triaging#up-voting-a-feature-request) the issue. If it receives 20 upvotes we will move it to our backlog. If not, we will close it. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by VSCodeTriageBot** [comment]
This feature request has not yet received the 20 community [upvotes](https://github.com/microsoft/vscode/wiki/Issues-Triaging#up-voting-a-feature-request) it takes to make to our backlog. 10 days to go. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by VSCodeTriageBot** [comment]
:slightly_frowning_face: In the last 60 days, this feature request has received less than 20 community upvotes and we closed it. Still a big Thank You to you for taking the time to create this issue! To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!
